exporter/alibabacloudlogserviceexporter/logsdata_to_logservice.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alibabacloudlogserviceexporter
+
+import (
+	"encoding/json"
+	"strconv"
+	"time"
+
+	sls "github.com/aliyun/aliyun-log-go-sdk"
+	"github.com/gogo/protobuf/proto"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
+)
+
+const (
+	slsLogTimeUnixNano   = "timeUnixNano"
+	slsLogSeverityNumber = "severityNumber"
+	slsLogSeverityText   = "severityText"
+	slsLogName           = "name"
+	slsLogContent        = "content"
+	slsLogAttribute      = "attribute"
+	slsLogFlags          = "flags"
+)
+
+func logDataToLogService(logger *zap.Logger, ld pdata.Logs) ([]*sls.Log, int) {
```

Comment:
Looks like this translation function ignores the Resource part of the log data. Is this intentional?

Reply:
It is not used by SLS now, but we may need Resource and InstrumentationLibrary in the future. We have added it.